src/qt/rpcconsole.cpp

```diff
@@ -1120,7 +1120,15 @@ void RPCConsole::updateNodeDetail(const CNodeCombinedStats *stats)
     ui->peerSubversion->setText(QString::fromStdString(stats->nodeStats.cleanSubVer));
     ui->peerDirection->setText(stats->nodeStats.fInbound ? tr("Inbound") : tr("Outbound"));
     ui->peerHeight->setText(QString::number(stats->nodeStats.nStartingHeight));
-    ui->peerWhitelisted->setText(stats->nodeStats.m_legacyWhitelisted ? tr("Yes") : tr("No"));
```

Comment:
Why wasn't `m_legacyWhitelisted` removed?

Reply:
See OP

>  This removes the one-but-last use of legacyWhitelisted.

Reply:
Oh right, miss interpreted "one-but-last", sorry.

Reply:
done in https://github.com/bitcoin/bitcoin/pull/20755 :)